server/database/schema.js

```diff
@@ -1,6 +1,7 @@
 import { MongoClient, ServerApiVersion } from 'mongodb';
 import 'dotenv/config';
 import { nanoid } from 'nanoid';
+import { insertERDiagramsData } from './dummyDataERD';
```

Comment:
It was right the first time. The way the project is setup, without specifying file endings here it will throw `ERR_MODULE_NOT_FOUND` error

```suggestion
import { insertERDiagramsData } from './dummyDataERD.js';
```